### ISSUE TITLE
SCM viewlet focus colors need work

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#21202]
#20999

You cannot tell the difference between an opened diff editor and a focused diff editor line in the viewlet:

Diff editor is opened and focused:

![image](https://cloud.githubusercontent.com/assets/2193314/23228696/580c9032-f8f2-11e6-8fe9-d5b20f546ade.png)

Diff editor line is focused:

![image](https://cloud.githubusercontent.com/assets/2193314/23228710/63526d7c-f8f2-11e6-9771-c0b21c17e2e2.png)

Clean button focused (needs more contrast):

![image](https://cloud.githubusercontent.com/assets/2193314/23228739/82ff023e-f8f2-11e6-86c0-7be8013ec724.png)

.travis.yml is focused, .gitignore is open (this looks like the opposite of what it should be to me):

![image](https://cloud.githubusercontent.com/assets/2193314/23228764/9b744b9e-f8f2-11e6-9859-f4eff125ff65.png)

---

For reference here's the explorer:

Item focused:

![image](https://cloud.githubusercontent.com/assets/2193314/23228778/aadd6c6e-f8f2-11e6-8dbd-012073748b72.png)

Editor focused:

![image](https://cloud.githubusercontent.com/assets/2193314/23228843/dc6e771e-f8f2-11e6-9a44-5aea11d012ac.png)

.travis.yml is focused, .gitignore is open (again this looks like the opposite of what it should be to me, cc @bpasero):

![image](https://cloud.githubusercontent.com/assets/2193314/23228856/ea9e2604-f8f2-11e6-9702-0505755b40c4.png)

**Comment by bpasero** [comment]
This is probably due to using the list widget vs tree widget that has different styles. +1 for same looks.

**Comment by joaomoreno** [comment]
The current list is now exactly the same as the tree. As for the _opposite_, we'll let theming solve it. 👍
